posthog/async_migrations/migrations/0004_replicated_schema.py

```diff
@@ -217,6 +225,25 @@ def move_partitions(self, from_table: str, to_table: str):
             sync_execute(f"ALTER TABLE {to_table} ATTACH PARTITION {partition} FROM {from_table}")
             sync_execute(f"ALTER TABLE {from_table} DROP PARTITION {partition}")
 
+    def rename_tables(
```

Comment:
we might want to have this in some sort of utility functions file for async migrations, but no need to do that now.